server/controllers/userAuth.js

```diff
@@ -448,6 +448,24 @@ const deleteAccount = async (req, resp) => {
   return resp.send();
 };
 
+const updateEmailNotificationSettings = async (req, resp) => {
+  const { _id } = req.user;
+  const { notificationFrequency } = req.body;
+
+  try {
+    await User.findOneAndUpdate(
+      { _id },
+      {
+        emailNotificationsFrequency: notificationFrequency
```

Comment:
Please be consistent - please use `emailNotificationFrequency` or `notificationFrequency` but pick one throught server/frontent.